src/app/scheduler/components/CourseCard.tsx

```diff
@@ -0,0 +1,92 @@
+import { AddIcon, CloseIcon, InfoOutlineIcon } from '@chakra-ui/icons';
+import { Box, Text, Flex, VStack, Checkbox, IconButton, Link } from '@chakra-ui/react';
+import { useCallback } from 'react';
+
+import { Term } from '../../../shared/fetchers';
+import { useSavedCourses } from '../../../shared/hooks/useSavedCourses';
+
+export type CourseCardProps = {
+  term: Term;
+  subject: string;
+  title: string;
+  code: string;
+  colour: string;
```

Comment:
This should be `color?: BackgroundProps['bg']`. Sadly the American spelling just to be consistent :\